src/boards/rainbow_esp.cpp

```diff
@@ -329,6 +342,8 @@ void BrokeStudioFirmware::processBufferedMessage() {
 					uint8 const n = this->rx_buffer[2];
 					this->readFile(this->working_path, this->working_file, n, this->file_offset);
 					this->file_offset += n;
+					UDBG("working file offset: %lu (%lx)\n", this->file_offset, this->file_offset);
```

Comment:
`this->file_offset` is an `uint32_t`, so it must be `%u` and `%x` instead of `%lu` and `%lx`

(Yes, printf-like printing is tricky as hell :( )